sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/output.py

```diff
@@ -80,14 +80,16 @@ def __init__(self, *, type=AssetTypes.URI_FOLDER, path=None, mode=None, descript
         self.mode = mode
         # use this field to determine the Output is control or not, currently hide in kwargs
         self.is_control = kwargs.pop("is_control", None)
+        # use this field to mark Output for early node orchestrate, currently hide in kwargs
+        self.early_available = kwargs.pop("early_available", None)
```

Comment:
Is there a doc explaining what do `is_control` and `early_available` mean?

Reply:
both of these fields are for pipelines only internal feature; to be more specific, `is_control` is for if-else condition and some other scenarios, `early_available` is for some upcoming features in pipelines.
if you are interested in the related pipelines feature, please ping me or @wangchao1230 , maybe some internal doc can kind of solve your confusion.